presto-hive-hadoop2/src/test/java/io/prestosql/plugin/hive/TestHiveFileSystemAbfsOAuth.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.hive;
+
+import io.prestosql.plugin.hive.azure.HiveAzureConfig;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Parameters;
+
+public class TestHiveFileSystemAbfsOAuth
+        extends AbstractTestHiveFileSystemAbfs
+{
+    private String clientEndpoint;
+    private String clientId;
+    private String clientSecret;
+
+    @Parameters({
+            "hive.hadoop2.metastoreHost",
+            "hive.hadoop2.metastorePort",
+            "hive.hadoop2.databaseName",
+            "test.hive.azure.abfs.container",
+            "test.hive.azure.abfs.storage-account",
+            "test.hive.azure.abfs.test-directory",
+            "hive.azure.abfs.oauth-client-endpoint",
```

Comment:
Add `test.` to all of these so that the names are consistent. Also, it's makes it clear these are test parameters and not system properties getting directly translated to config properties.

Reply:
Here I was using the property names from `HiveAzureConfig` for the properties that are also present there. But I agree that the inconsistency is more confusing; fixed.